### PR TITLE
PP-11756 Update page as digital wallets are now live

### PR DESCRIPTION
**Pull request by alanmaddrell**
Removed outdated references and tidied up text for style

### CODE REVIEW

File: source/apple-pay-and-google-pay.html.erb [hunk 1]
```diff
@@ -25,10 +25,10 @@ title: Apple Pay and Google Pay
           on GOV.UK&nbsp;Pay.
         </p>
 
-        <h2 class="govuk-heading-m">How Apple Pay and Google Pay works</h2>
+        <h2 class="govuk-heading-m">How Apple Pay and Google Pay work</h2>
 
         <p class="govuk-body">Any user with Apple Pay or Google Pay enabled can make payments on your service in seconds.</p>
-        <p class="govuk-body">Because Apple Pay and Google already have users details, they don’t have to fill them in again on GOV.UK&nbsp;Pay. They simply have to:</p>
+        <p class="govuk-body">Because Apple Pay and Google already have users' details, they do not have to fill them in again on GOV.UK&nbsp;Pay. They just have to:</p>
```
Comment (alexbishop1) [inline source/apple-pay-and-google-pay.html.erb:31]: Can this apostrophe (and the others) be curly — so `’` rather than `'`?

**Review by alexbishop1 — CHANGES_REQUESTED** [review]
Content looks great. One thing about the punctuation. It looks the linter does not like the way the file is formatted — a developer should be able to help you fix that.

File: source/apple-pay-and-google-pay.html.erb (hunk 1 above)
Comment (alanmaddrell) [reply source/apple-pay-and-google-pay.html.erb]: I've hopefully fixed that now. Is that everything?

**Review by alexbishop1 — DISMISSED** [review]
I’m happy but the linter isn’t. Grab a developer (who has a development environment set up!) to help figure that out.

**Review by stephencdaly — APPROVED** [review]

**Comment by stephencdaly** [comment]
@alexbishop1 I've fixed the build issue. Could you re-approve please

**Review by james-peacock-gds — APPROVED** [review]
LGTM

**Pull request merged**